inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleTableSourceFactory.java

```diff
@@ -95,6 +95,12 @@ public class OracleTableSourceFactory implements DynamicTableSourceFactory {
                             "Optional startup mode for Oracle CDC consumer, valid enumerations are "
                                     + "\"initial\", \"latest-offset\"");
 
+    public static final ConfigOption<Boolean> MIGRATE_ALL =
+            ConfigOptions.key("migrate-all")
```

Comment:
Maybe it is better to call 'source.multiple.enable'?

Reply:
done.